.gitattributes

```diff
@@ -1 +1,2 @@
-* text=auto
\ No newline at end of file
+* text=auto
```

Comment:
Can we add here more, like for example:

# Set the default behavior to automatically normalize line endings.
* text=auto

# Explicitly declare text files you want to always be normalized and converted to LF upon checkout.
*.c text eol=lf
*.cpp text eol=lf
*.h text eol=lf
*.txt text eol=lf
*.json text eol=lf
*.yml text eol=lf
*.md text eol=lf

# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary

Reply:
I can include that - but I did not done that because this would lead to massive line change upon next checkout && commit of the respective files(if they differ). See my last comment https://github.com/Azure/azure-osconfig/pull/810/files#r1846321012

Reply:
Thanks! See my reply to your explanation there. I agree with your proposal.

Reply:
Resolving see my update in https://github.com/Azure/azure-osconfig/pull/810#discussion_r1848626987. We should continue discussion there